packages/ps/test/helpers/init.js

```diff
@@ -1,2 +1,8 @@
 const path = require('path')
 process.env.TS_NODE_PROJECT = path.resolve('test/tsconfig.json')
+
+let nock = require('nock')
+nock.disableNetConnect()
+if (process.env.ENABLE_NET_CONNECT === 'true') {
+  nock.enableNetConnect()
+}
```

Comment:
This block and similar seem like a good if/else use case.

Reply:
I thought about that, but I'm trying to make absolutely sure that network connections are disabled for unit tests.